viadot/sources/mindful.py

```diff
@@ -27,7 +28,7 @@ def __init__(
         """Mindful connector which allows listing and downloading into Data Frame or specified format output.
 
         Args:
-            header (str): Header with credentials for calling Mindful API.
+            auth (Tuple[str]): Authentication credentials for calling Mindful API. The structure is user and password.
```

Comment:
The docstring says that `The structure is user and password`. In the task, you are getting `CUSTOMER_UUID` and `AUTH_TOKEN`. It doesn't look like a username and password.

Reply:
But the auth accepts the function `HTTPBasicAuth` which only needs the user and password.
I  you think it necessary for name consistency, I could replace `user` and `password` in docstring by `customer_uuid` and `auth_token` respectively.